<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageActionSelect</class>
 <widget class="QWizardPage" name="WizardPageActionSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QRadioButton" name="radioExisting">
     <property name="text">
      <string>Existing Content</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioCustom">
     <property name="text">
      <string>Custom Script</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <widget class="QTextEdit" name="scriptEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
